fuel-core/src/args.rs

```diff
@@ -53,9 +53,15 @@ pub struct Opt {
     /// The minimum allowed gas price
     #[clap(long = "min-gas-price", default_value = "0")]
     pub min_gas_price: u64,
+
     /// The minimum allowed byte price
     #[clap(long = "min-byte-price", default_value = "0")]
     pub min_byte_price: u64,
+
+    /// Enable predicate execution on transaction inputs.
+    /// Will reject any transactions with predicates if set to false.
+    #[clap(long = "predicates")]
+    pub predicates: bool,
```

Comment:
contract creation currently rejects tx create with invalid predicates. should we disable that default check?

https://github.com/FuelLabs/fuel-vm/blob/ffaf70c6d0f7974de4bd316c1742f3698ec8c219/src/interpreter/executors/main.rs#L132-L137

Reply:
yes probably, triggering those checks should be outside the VM (e.g. from the txpool)

Reply:
not a blocker atm but could you file a tech debt issue?